app/models/commodity.rb

```diff
@@ -153,4 +144,23 @@ def changes(depth = 1)
      .limit(TradeTariffBackend.change_count)
      .order(Sequel.desc(:operation_date, nulls: :last), Sequel.desc(:depth))
   end
+
+  private
+
+  def ancestor_indents
```

Comment:
This returns a non-executed dataset of indents that we join against in the ancestors method. I've made 0 changes to this behaviour, I just moved it to demark responsibilities for distinct parts of the query - one returns commodities, one returns indents - and to make the code slightly more readable.